### ISSUE TITLE
RegionLock for items

### ISSUE THREAD

**Issue by clery** [LockManipulator/Cubegression-Light#1]
Hi there,

I've been trying to get into modding (and I don't get half the stuff going on in the code), so I compiled your stuff to remove region lock for movement items, but for some reason, I get to move with my mount in a new zone, but not with the glider or anything else.

Any idea why it's doing that ?

**Comment by LockManipulator** [comment]
This mod already removes region locking from movement items (boat, reins, and glider). Just make sure that your config.txt starts with "movement=true" (without quotes) on the first line.

**Comment by clery** [comment]
Yeah I know, I did that, and it just doesn't seem to work as expected for me. I must've done something wrong, maybe when compiling
